test/e2e/client/restclient.go

```diff
@@ -100,14 +102,18 @@ func NewFireFly(l Logger, hostname, namespace string) *FireFlyClient {
 func NewResty(l Logger) *resty.Client {
 	client := resty.New()
 	client.OnBeforeRequest(func(c *resty.Client, req *resty.Request) error {
-		l.Logf("==> %s %s %s", req.Method, req.URL, req.QueryParam)
+		var b []byte
+		if _, isReader := req.Body.(io.Reader); !isReader {
+			b, _ = json.Marshal(req.Body)
+		}
+		l.Logf("%s: ==> %s %s %s: %s", fftypes.Now(), req.Method, req.URL, req.QueryParam, string(b))
```

Comment:
Feels a little odd that we add timestamps inside these logs (shouldn't that be a higher-level job delegated to the logging framework for all logs?)

Reply:
Yes, but I couldn't find an option in the Go test framework to make `t.Logf` add timestamps... so this is a workaround that didn't mean changing every log line